src/GafferScene/CopyOptions.cpp

```diff
@@ -0,0 +1,98 @@
+#include "GafferScene/CopyOptions.h"
+#include "Gaffer/StringAlgo.h"
+
+#include "boost/algorithm/string/predicate.hpp"
+
+// using namespace IECore;
+using namespace Gaffer;
+using namespace GafferScene;
+
+IE_CORE_DEFINERUNTIMETYPED( CopyOptions );
+
+size_t CopyOptions::g_firstPlugIndex = 0;
+
+CopyOptions::CopyOptions( const std::string &name )
+	:	GlobalsProcessor( name )
+{
+	storeIndexOfNextChild( g_firstPlugIndex );
+	addChild( new ScenePlug( "sourceScene", Plug::In ) );
```

Comment:
Let's just call this "source", since the existing Parent and Instancer nodes use the shorter form for their secondary scene plugs ("child" and "instance"), and we use just "in" for the primary scene.